.github/workflows/ci-nightly-fips.yaml

```
@@ -0,0 +1,194 @@
+
+      - name: Setup Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: "~1.24"
+
+      - name: Tidy go.mod files
+        run: go mod tidy
+
+      - name: Verify build
+        run: make ci DISTRIBUTIONS=${{ matrix.distribution }}
+
+      - name: Login to Docker
+        uses: docker/login-action@v3
+        if: ${{ env.ACT }}
+        with:
+          registry: docker.io
+          username: ${{ secrets.OTELCOMM_DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.OTELCOMM_DOCKER_HUB_PASSWORD }}
+
+      - uses: docker/setup-qemu-action@v2
+
+      - uses: docker/setup-buildx-action@v2
+
+      - name: Import GPG key
+        id: import_gpg
+        uses: crazy-max/ghaction-import-gpg@v6
+        with:
+          gpg_private_key: ${{ secrets.OTELCOMM_GPG_PRIVATE_KEY_BASE64 }}
+          passphrase: ${{ secrets.OTELCOMM_GPG_PASSPHRASE }}
+
+      - name: Write GPG to path in memory for signing rpm/deb
+        id: write_gpg_to_path
+        run: |
+          GPG_KEY_PATH="$(mktemp /dev/shm/gpg.XXXXXX)"
+          echo "$GPG_PRIVATE_KEY" | base64 -d >> "$GPG_KEY_PATH"
+          echo "gpg_key_path=$GPG_KEY_PATH" >> $GITHUB_OUTPUT
+        env:
+          GPG_PRIVATE_KEY: ${{ secrets.OTELCOMM_GPG_PRIVATE_KEY_BASE64 }}
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-access-key-id: ${{ secrets.OTELCOMM_AWS_TEST_ACC_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.OTELCOMM_AWS_TEST_ACC_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+          role-to-assume: arn:aws:iam::${{ secrets.OTELCOMM_AWS_TEST_ACC_ACCOUNT_ID }}:role/resource-provisioner
+          role-skip-session-tagging: true
+
+      - name: Login to ECR
+        uses: docker/login-action@v3
+        with:
+          registry: ${{ env.REGISTRY}}/${{ matrix.distribution }}
+
+      - name: Build and publish ${{ matrix.distribution }} nightly fips binaries & packages with GoReleaser
+        id: goreleaser_publish
+        if: ${{ !env.ACT }}
+        uses: goreleaser/goreleaser-action@v6
+        env:
+          NFPM_PASSPHRASE: ${{ secrets.OTELCOMM_GPG_PASSPHRASE }}
+          GPG_FINGERPRINT: ${{ steps.import_gpg.outputs.fingerprint }}
+          GPG_KEY_PATH: ${{ steps.write_gpg_to_path.outputs.gpg_key_path }}
+          REGISTRY: ${{ env.REGISTRY }}
+        with:
+          distribution: goreleaser
+          version: '2.11.2'
+          args: --skip=announce,validate --clean --timeout 2h --config .goreleaser-nightly-fips.yaml
+          workdir: distributions/${{ matrix.distribution }}
+
+      - name: Extract Docker Manifest SHA
+        run: |
+          docker_manifest_sha=$(echo '${{ steps.goreleaser_publish.outputs.artifacts }}' |
+            yq -r '.[] | select(.type == "Docker Manifest") | .extra.Digest')
+            jq -r '.[] | select(.type == "Docker Manifest") | select(.name | endswith(":nightly-fips")) | .extra.Digest')
```

Fat-fingered my suggestion, should've deleted the yq line and just keep the updated jq line...
```suggestion
            jq -r '.[] | select(.type == "Docker Manifest") | select(.name | endswith(":nightly-fips")) | .extra.Digest')
```